### DISCUSSION TITLE
It just doesn't work for me

### DISCUSSION THREAD

**Discussion by Alonzzzo2** [manutalcual/winredock#42] [Q&A]
Placed the app at localappdata
Ran it
placed all the windows at the right location on the external monitor
disconnected the external monitor, reconnected again, no window was relocated to the right location.

What am I doing wrong?

**Comment by manutalcual** [comment]
Which branch are you compiling?

**Reply by Alonzzzo2** [reply]
I'm not compiling, as per instructions, I've just downloaded bin/bin.zip, extracted, used it

**Reply by manutalcual** [reply]
Ok, We have more recent code. I'll compile and create a new binary and notify here to you.

**Reply by Alonzzzo2** [reply]
That will be great, thanks!

**Comment by Bretterson** [comment]
Same issue here.

**Reply by Bretterson** [reply]
I just tried switching to the logging version. I've disconnecting and reconnecting to my dock a few times. For some of those attempts I opened the laptop lid to make it change to one window and for some I just reconnected after a 30 seconds or so without opening the laptop lid... so far it has only worked once.
